legacyClaim/starknet/src/claims.cairo

```diff
@@ -0,0 +1,3006 @@
+fn claims_mapping(num: felt252) -> (felt252, felt252) {
+    match num {
+        0 => panic!("zero claims"), 
+        1 => (0xDB65702A9b26f8a643a31a4c84b9392589e03D7c, 0xD3C21BCECCEDA1000000),
```

Comment:
The values don't look right. It's the same owner addr and amount, repeated 3000 times.

Reply:
they are placeholder